gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetFinder.java

```diff
@@ -21,19 +21,25 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
+import java.util.Optional;
 import java.util.Properties;
 
 import org.apache.commons.lang.StringUtils;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 
+import com.google.api.client.util.Maps;
```

Comment:
is it possible to use new HashMap<>() and avoid this dep?

Reply:
yeah sure! Pushed the changes in my latest commit

Reply:
Still see this dep? You can remove it if it's a dead import

Reply:
my bad... removed it!